neuralmonkey/nn/projection.py

```diff
@@ -64,3 +67,17 @@ def maxout(inputs, size, scope="MaxoutProjection"):
 
         reshaped = tf.reshape(maxpooled, [-1, size])
         return reshaped
+
+
+def multilayer_projection(input_, layer_sizes, activation=tf.nn.relu,
```

Comment:
sice je to projekce, ale je to perceptron a patří to podle mě jinam. Taky tenhle kód dělá to samý co ta třída, takže dávat to jinam ještě víc zamaskuje, že máme duplicitní kód.

Btw. pořád jsem přesvědčen, že tahle funkce je to, co by se mělo používat (a ne ta třída)